invenio_accounts/hash.py

```diff
@@ -61,26 +61,42 @@ def _mysql_aes_key(key):
 
 
 def _mysql_aes_pad(val):
-    """Padding."""
+    """Padding.
+
+    .. note:: Useful only for MySQL.
```

Comment:
@hachreak the note is obvious from the function name. Can you remove it for functions prefixed with `_?mysql`?